javascript/core/array-methods/shuffle-an-array-using-sort-and-random.md

```diff
@@ -35,68 +35,96 @@ links:
 Given the array:
 
 ```javascript
-var array = [1,2,3,4,5,6,7,8,9];
+let array = [1, 2, 3, 4, 5, 6, 7, 8, 9];
 ```
 
-Sort randomly with:
+You could sort it randomly using `Array.prototype.sort()`[1]:
 
 ```javascript
-array.sort(function() {
+array.sort((a, b) => {
   return Math.random() - 0.5
 });
+// The order will be different
+// every time
+// [5, 4, 2, 1, 6, 3, 7, 8, 9]
 ```
 
-The `- 0.5` means the returned value exists between the range `-0.5 < x < 0.5`, allowing the sort function to decide where to position the next item.
+This is due to how the `Array.prototype.sort()` method works. The arrow function we pass as an argument will return a number, and depending on its value it will sort the array:
+- 0 -> `a = b`
+- > 0 -> `a > b`
+- < 0 -> `b > a`
```

Comment:
And then, in combination with the [suggestion above](https://github.com/enkidevs/curriculum/pull/1753/files#r358801519), we add:

- 0 -> sort order doesn't change
- > 0 -> `b` comes before `a`
- < 0 -> `a` comes before `b`